src/Components/ProfileUserBanner.js

```diff
@@ -85,6 +86,7 @@ export default function ProfileUserBanner() {
               fontSize: { xs: "1.66rem", md: "2.5rem" },
             }}
           />
+          <Typography variant="h4">@{profile?.handle}</Typography>
```

Comment:
do we want an "@" sign to show up for users without a handle? or for this to just be blank?

Reply:
Great catch!!! @ symbol has been removed if there's no handle to display